tensorflow/tools/ci_build/Dockerfile.cpu

```diff
@@ -13,3 +13,11 @@ RUN /install/install_bazel.sh
 # Set up bazelrc.
 COPY install/.bazelrc /root/.bazelrc
 ENV BAZELRC /root/.bazelrc
+
+# Script for (optional) test on install
+COPY builds/test_on_install.sh /builds/test_on_install.sh
+RUN chmod a+rx /builds/test_on_install.sh
+
+# Grant users installation permission on Python package directory
+RUN chmod -R a+rw /usr/local/lib/python2.7/dist-packages
```

Comment:
Fixed as suggested. I now use pip "install --user" to install the whl file in the user's home folder.